### PR TITLE
Feat/ru_labeling

### PR DESCRIPTION
**Pull request by lsig**
# Script for getting data from label studio
In this PR request I made a script that gets data from label studio and formats it in the correct way so our model can train on the data. The script is type annotated and required me to add a dotenv dependency for loading the the api key from the .env file. 

# Testing 
- Ran the script on local machine with my api_key which produced the desired output 
- Ran format_and_lint before committing 
- No unit tests

# Bugs 
- Mypy complains: Incompatible types (expression has type "MetaInfo", TypedDict item "meta" has type "MetaInfo") in line 28 even though the meta item only has the type MetaInfo, but nonetheless the script runs as expected.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+__pycache__/
```
Comment (Eysteinn-Orn) [inline .gitignore:1]: We probably should add this to the repository, possibly could be its own PR. But this is also good!

File: training_datasets/ruquad_labeling.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    all_project_records = []
+
+    for i in PROJECT_IDS:
+        p_records = __records(i, key)
+        all_project_records.extend(p_records)
+
+    return all_project_records
+
+
+def __invalid_record(record: CleanRUResponse) -> bool:
```
Comment (Eysteinn-Orn) [inline training_datasets/ruquad_labeling.py:43]: A better name for this would be __is_valid_record to indicate that this is a boolean function more-so than just the type hint

File: training_datasets/ruquad_labeling.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+PROJECT_IDS = [1, 4, 5]
+
+
+def __api_key() -> str:
+    env_vars = dotenv_values()
+    api_key: str = env_vars.get("API_KEY", "")
+    return api_key
+
+
+def __records(project_id: int, api_key: str) -> list[CleanRUResponse]:
```
Comment (Eysteinn-Orn) [inline training_datasets/ruquad_labeling.py:18]: This could also be renamed to __get_records or __get_clean_records. This is optional.

File: training_datasets/ruquad_labeling.py (hunk 2 above)
Comment (Eysteinn-Orn) [reply training_datasets/ruquad_labeling.py:43]: I think you can refactor this function to be more readable;
Maybe something like this:
```python
def __is_invalid_record(record: CleanRUResponse) -> bool:
    annotations = record["annotations"]
    if not annotations or not annotations[0]["result"]:
        return True

    result = annotations[0]["result"][0]["value"]
    if "labels" not in result or result["labels"][0] == "Archive":
        return True

    paragraph = record["meta"]["paragraph"]
    if len(paragraph.split(" ")) > 300:
        return True

    return False
```

**Review by Eysteinn-Orn — APPROVED** [review]
Overall, looks good and a good contribution!

File: training_datasets/ruquad_labeling.py (hunk 2 above)
Comment (lsig) [reply training_datasets/ruquad_labeling.py:43]: I feel like the context of the function (where it is used) makes it obvious. if it is an invalid record, then we skip it.

**Review by njallskarp — APPROVED** [review]
very nice. I like

**Review by Eysteinn-Orn — APPROVED** [review]
